packages/magento-wishlist/components/ProductWishlistChip/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { IconSvg, iconHeart, extendableComponent } from '@graphcommerce/next-ui'
+import { SxProps, Theme, IconButton, IconButtonProps } from '@mui/material'
+import React, { useState, useEffect } from 'react'
+import { useQuery, useMutation } from '@apollo/client'
+import {
+  AddProductToWishlistDocument,
+  RemoveProductFromWishlistDocument,
+} from '@graphcommerce/magento-wishlist'
+import { CustomerTokenDocument } from '@graphcommerce/magento-customer'
+import {
+  GUEST_WISHLIST_STORAGE_NAME,
+  GetIsInWishlistsDocument,
+} from '@graphcommerce/magento-wishlist'
+import { ProductWishlistChipFragment } from './ProductWishlistChip.gql'
+import { t } from '@lingui/macro'
+
+type ProductWishlistSettings = {
+  display?: 'guest' | 'customer'
+}
+
+export type ProductWishlistChipProps = ProductWishlistChipFragment &
+  IconButtonProps &
+  ProductWishlistSettings & { sx?: SxProps<Theme> }
+
+const name = 'ProductWishlistChip' as const
+const parts = ['root', 'iconHeart', 'iconHeartActive', 'wishlistButton'] as const
+const { classes } = extendableComponent(name, parts)
+
+export default function ProductWishlistChip(props: ProductWishlistChipProps) {
+  const { display, sku, sx = [] } = props
+
+  const [inWishlist, setInWishlist] = useState(false)
+  const [displayWishlist, setDisplayWishlist] = useState(true)
+
+  const { data: token } = useQuery(CustomerTokenDocument)
+  const isLoggedIn = token?.customerToken && token?.customerToken.valid
+
+  const heart = (
+    <IconSvg
+      src={iconHeart}
+      size='medium'
+      className={classes.iconHeart}
+      sx={(theme) => ({ stroke: '#AC2E2E' })}
```

Comment:
- Use color attribute to define color, not stroke. (IconSvg has `stroke: currentColor`)
- Use a color from the theme.palette object
- Isn't theme.palette.text.primary the better solution here? Or: theme.palette.text.secondary